pkg/config/expand.go

```diff
@@ -545,3 +549,43 @@ func intersection(s1 []string, s2 []string) []string {
 	slices.Sort(is)
 	return is
 }
+
+// Find the closest module that the reference could be referring to (only ones that are defined before reference)
+// Returns a string with the closest module name (default vars)
+// Currently limited to finding strings that are at least 40% similar based on module string length and levenshtein distance
+func closestReference(bp *Blueprint, mod *Module, ref *Reference) string {
```

Comment:
This function doesn't look for reference but rather closes module_id(+ "vars"), let's name it appropriately, e.g. "hint module id fix"